Sources/Sentry/SentryUIViewControllerSwizziling.m

```diff
@@ -43,6 +44,75 @@ + (void)swizzleViewControllerInits
         }),
         SentrySwizzleModeOncePerClassAndSuperclasses, swizzleViewControllerInitWithNib);
+
+    [SentryUIViewControllerSwizziling swizzleRootViewController];
+}
+
+/**
+ * If an app targets, for example, iOS 13 or lower, the UIKit inits the initial/root view controller
+ * before the SentrySDK is initialized. Therefore, we manually call swizzle here not to lose
+ * auto-generated transactions for the initial view controller. As we use
+ * SentrySwizzleModeOncePerClassAndSuperclasses, we don't have to worry about swizzling twice. We
+ * could also use objc_getClassList to lookup sub classes of UIViewController, but the lookup can
+ * take around 60ms, which is not acceptable.
+ */
++ (void)swizzleRootViewController
+{
+    if (![UIApplication respondsToSelector:@selector(sharedApplication)]) {
+        [SentryLog logWithMessage:@"UIApplication doesn't respont to sharedApplication. Skipping "
+                                  @"swizzleRootViewController."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    UIApplication *app = [UIApplication performSelector:@selector(sharedApplication)];
+
+    if (app == nil) {
+        [SentryLog logWithMessage:@"UIApplication is nil. Skipping swizzleRootViewController."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    if (app.delegate == nil) {
+        [SentryLog
+            logWithMessage:@"UIApplicationDelegate is nil. Skipping swizzleRootViewController."
+                  andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    // Check if delegate responds to window, which it doesn't have to.
+    if (![app.delegate respondsToSelector:@selector(window)]) {
+        [SentryLog logWithMessage:@"UIApplicationDelegate doesn't respond to window selctor. "
+                                  @"Skipping swizzleRootViewController."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    if (app.delegate.window == nil) {
+        [SentryLog logWithMessage:
+                       @"UIApplicationDelegate.window is nil. Skipping swizzleRootViewController."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    UIViewController *rootViewController = app.delegate.window.rootViewController;
+    if (rootViewController == nil) {
+        [SentryLog logWithMessage:@"UIApplicationDelegate.window.rootViewController is nil. "
+                                  @"Skipping swizzleRootViewController."
+                         andLevel:kSentryLevelDebug];
+        return;
+    }
+
+    NSArray<UIViewController *> *allViewControllers = rootViewController.descendantViewControllers;
+
+    for (UIViewController *viewController in allViewControllers) {
+        Class viewControllerClass = [viewController class];
+        if (viewControllerClass != nil) {
+            [SentryLog logWithMessage:@"Calling swizzleRootViewController."
```

Comment:
l: I think it would be nice to have the name of the class here in the log being swizzled.